src/foundry/client/pixi/extensions/observable-transform.d.mts

```diff
@@ -0,0 +1,30 @@
+export {}
+
+declare global {
+  /**
+   * A custom Transform class allowing to observe changes with a callback.
+   * @privateRemarks Scope extends object is intentional; `"The scope bound to an ObservableTransform class must be a valid object/class."`
+   */
+  class ObservableTransform<Scope extends object, CB extends (this: Scope) => unknown> extends PIXI.Transform {
```

Comment:
it'd probably be better to be `Scope extends AnyObject`. I don't see a compelling reason to take functions, classes, or arrays here.

Reply:
Classes seem like a valid use case, as the `onChange` references `this.cb.call(this.scope);`